organizational/conduct.rst

```diff
@@ -0,0 +1,101 @@
+The Mission of Hacker Society
+=============================
+
+A time and place for people to learn about technology from each other, in order
+to build a community, to empower the individual, promote equality, and build an
+open society.
+
+**Open Society**: An open society promotes freedom of thought and expression,
+personal responsibility, individualism, cooperation, and equality of all
+citizens.
+
+Code of Conduct
+===============
+
+It is expected that members behave in a way that allows Hacker Society to
+achieve its mission.  Participants in Hacker Society events are expected to
+treat each other with dignity and respect for each other's perspectives,
+experiences, and knowledge.  In particular, participants should recognize that
+people of all skill levels are welcome, and should always be willing to help
+others learn.
+
+To support Hacker Society's mission of: learning technology from each other,
+building community, and promoting equality, Hacker Society must be a place where
+people can be comfortable regardless of gender, gender identity and expression,
+sexual orientation, disability, physical appearance, body size, race, age, or
+religion. Therefore, we do not tolerate:
+
+- Rude or unwelcome comments about a person's gender, gender identity, gender
+  expression, sexual orientation, disability, physical appearance, attire, race,
+  age or religious expression.
+- Sexual images in public spaces
+- Deliberate intimidation, stalking, or following
+- Harassing photography or recording
+- Sustained disruption of talks or other events
+- Inappropriate physical contact
+- Unwelcome sexual attention
+- Advocating for, or encouraging, any of the above behavior
+
+Necessarily, the goals of learning, community, and equality are in tension
```

Comment:
> My reasoning: we have had people in hacsoc who have been made rude comments but have also been valuable members of the community in other ways. College is a time when many people push boundaries on what is polite and acceptable.

I should note: the people and comments I am thinking of would not have violated the current code as written. They were rude but they were not attacking a persons intrinsic characteristics. As I have said there is a line in the sand. We should be clear where it is so that these people can also participate. The world would be the poorer without them.